ProcessSlave.php

```diff
@@ -258,7 +258,7 @@ public function run()
     {
         $this->loop = \React\EventLoop\Factory::create();
 
-        $this->errorLogger = new BufferingLogger();
+        $this->errorLogger = BufferingLogger::create();
         ErrorHandler::register(new ErrorHandler($this->errorLogger));
```

Comment:
Only in symfony/debug >= 2.8. The __construct() method expecting \Symfony\Component\Debug\BufferingLogger was added in 2.8: https://github.com/symfony/debug/commit/be37182729ff968d9918f4cbbf88a0d5349dcb5a#diff-b91ca53d6ab50b804cb5783aaf1f9740 . 

So, if you are running symfony/debug 2.7, passing PHPPM\Debug\BufferingLogger will be OK.